tests/unit_tests/utest_main.cpp

```diff
@@ -0,0 +1,19 @@
+/** \file utest_main.cpp
+ *  Entry point for unit tests
+ */
+
+#include "gtest/gtest.h"
+#include "OpenTurbineTestEnv.H"
+
+//! Global instance of the environment (for access in tests)
+openturbine_tests::OTurbTestEnv* utest_env = nullptr;
+
+int main(int argc, char** argv)
```

Comment:
We need to call Kokkos::initialize before we can run any unit tests using Kokkos constructs and finalize after all tests are run (you can only call each one once).  This is most easily done in the main function.

Reply:
Yes good catch, I've fixed this. The primary infrastructure is here for the unit testing, but this should be modified into a form that is more appropriate for the architecture of the software when that is decided.